.github/pull_request_template.md

```diff
@@ -1,61 +1,36 @@
 <!--
-- - -  I N S T R U C T I O N S  --  P L E A S E  R E A D  - - -
-Please remove boiler plate instructions when filling out this template.
-Please remove unwanted/unrelated/irrelevant information such as comments provided as a reference in this template.
-Use proper formatting to separate code snippets from text description e.g. use ```code block``` or `in-line code`.
-Please copy any output files into a Github gist (for e.g.) and link to the gist, rather than relying on paths on platforms that might change or disappear.
-
-Before opening a PR, please note these guidelines:
-
-- Each PR should only address ONE topic and have an associated GitHub issue
-- No hardcoded or paths to personal directories should be present
-- No temporary or backup files should be committed
-- Any code that was disabled by being commented out should be removed
-- Please provide a on-line, but descriptive title to the PR e.g. Adds XYZ functionality or Fixes ABC bugs, etc.  DO NOT use branch names as titles.
--->
 
-**Description**
+INSTRUCTIONS
 
-<!-- Please include relevant motivation and context. -->
-<!-- Please include a summary of the change and which issue is fixed. -->
-<!-- List any dependencies that are required for this change. -->
+- Please do not commit temporary, backup, or binary files.
+- Please remove commented-out code.
+- Please ensure code, config files, etc., contain no hardcoded paths.
+- Please format code snippets in PR description/comments with ```code block``` or `inline code`.
+- Please consider adding your own review comments to guide other reviewers.
+
+-->
 
-<!-- Please provide reference to the issue this pull request is addressing. -->
-<!-- For e.g. Fixes #IssueNumber -->
+**Description**
 
-**Type of change**
+<!-- A summary of the change, including relevant motivation, context, useful links, etc. -->
 
-Please delete options that are not relevant.
+**Type**
 
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-- [ ] This change requires a documentation update
+Select one or more:
 
-**How Has This Been Tested?**
+- [ ] Bug fix (corrects a known issue)
+- [ ] Code maintenance (refactoring, etc.)
+- [ ] New feature (adds functionality)
```

Comment:
Does this accurately capture the kinds of updates we make? In particular, the `Code maintenance` option seems to me to describe many PRs: It improves the code base without either adding functionality or fixing issues.

Reply:
Hm, the other question is whether this is redundant with the git labels - which we definitely should be more strict about if we cut this out

Reply:
That's interesting. If we wanted to go that way, we could replace this list with an `Affirm` checkbox stating that you've set the label to one of, say, three values. That seems like a good idea if we can get team buy-in.

Reply:
I thought of a couple more potential categories (because I was wondering how _this_ PR might be tagged):

- Documentation updates
- Tooling (CI, packaging, revision control, code-quality tools, etc.)

Reply:
It occurred to me: Replacing this checklist with a request that submitters tag the PR would presumably depend on their rights to add labels. @WeirAE I reopened https://github.com/maddenp-noaa/uwtools/pull/1 -- can you see if you can add labels to it?

Reply:
Oh, true - Nope, can't add anything there, that would be a problem!

Reply:
It was a good idea, but probably not a substitute for this checklist. I'll add my other two items here and we can keep the checklist.